ocaml/typing/printtyp.ml

```diff
@@ -845,7 +845,14 @@ let is_unambiguous path env =
       List.for_all (fun p -> lid_of_path p = id) rem &&
       Path.same p (fst (Env.find_type_by_name id env))
 
-let rec get_best_path r =
+let ambiguity_penalty path env =
+  if is_unambiguous path env then 0 else 10
+
+let path_size path env =
+  let l, s = path_size path in
```

Comment:
I think it would be worth moving `path_size` and `penalty` down to join these two functions.

Reply:
Done.